src/components/Book.css

```diff
@@ -2,60 +2,82 @@
   min-height: 150px;
   display: grid;
   grid-template-columns: 45% 30% 30%;
+  align-items: center;
   background-color: #fff;
   padding: 24px;
   margin: 24px 0;
   border-radius: 5px;
 }
 
 .progress {
-  width: 70px;
   height: 70px;
+  display: flex;
+  align-items: center;
+  width: 250px;
+  font-family: 'Montserrat', sans-serif;
 }
 
 .book p {
   margin: 0;
-  color: gray;
-}
-
-.book-col h3 {
-  margin: 0;
 }
 
 .book-col .category {
-  font-size: 14px;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
   margin-bottom: 3px;
   font-weight: 600;
+  color: gray;
 }
 
 .book-col button {
   padding: 0;
   width: 80px;
   height: 30px;
-  color: gray;
   background-color: #fff;
   border-radius: 0;
+  font-size: 15px;
+  font-weight: 400;
+  color: #4386bf;
+  font-family: 'Roboto Slab', serif;
+}
+
+.book h3 {
+  font-size: 1.375rem;
+  color: gray;
+}
```

Comment:
- [ ] please use a black color for the book title, as précised by the previous reviewer. kindly Try as much as possible to follow the original design.
- [ ] Great job implementing some of the changes. Please, like the previous reviewer noted, kindly ensure your buttons follow the original design color. Please use this color for the buttons `#0290ff`
yours
![Screenshot_2](https://github.com/wacoo/react-book-store/assets/65298048/7ecdc7a1-a1ea-431f-9914-62a66a0df665)
original
![Screenshot_1](https://github.com/wacoo/react-book-store/assets/65298048/0fa2bb21-e795-4a54-b70e-b5f28e3e3464)


- [optional] please kindly ensure your add book button follows the size of the original design.

Reply:
> - [ ] As the previous reviewer requested, it seems you haven't used black color for the book title. Please add that, as it will help you pay closer attention to details. 👍